test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -296,6 +296,46 @@ function ExecuteWebRequest
     return $result
 }
 
+# Starts the ClientCertificateCheck listener
+function StartClientCertificateCheck 
```

Comment:
If we move the function in ` HelpersCommon.psm1` it is ok rename it to ` Start-ClientCertificateCheck`.